Prefetching-Library/android_prefetching_lib/src/main/java/nl/vu/cs/s2group/nappa/prefetch/GreedyWithPageRankScoresPrefetchingStrategy.java

```diff
@@ -29,6 +29,10 @@ public GreedyWithPageRankScoresPrefetchingStrategy(float threshold) {
         this.threshold = threshold;
     }
 
+    @Override
+    public boolean needVisitTime() {
+        return false;
+    }
```

Comment:
These are only needed when not extending the class `AbstractPrefetchingStrategy`. However, to extend this class we need to do a small refactoring, which is outside the context of this PR